admin/templates/applications/application.html

```diff
@@ -1,10 +1,11 @@
 {% macro adminapplication(application_data) %}
-    <h1>
+    <h1 style="font-size: 24px;">
```

Comment:
We have two PDF bundle templates:
* `admin/templates/applications/application.html` (this file)
* `grc/templates/applications/application.html`

There's a few places where we haven't kept these in sync.

I've raised a PR to put the templates back in sync and to add a unit test to check that they stay in sync: https://github.com/cabinetoffice/grc-app/pull/169